docs/Configuration.md

```diff
@@ -85,19 +85,50 @@ Note that certain chain options affect the format and indexing of the chain data
 ## HTTP
 
 - `http-host`: HTTP host to listen on (default: 127.0.0.1).
-- `http-port`: HTTP port to listen on (default: 8332).
+- `http-port`: HTTP port to listen on (default: 8332 for mainnet).
 - `ssl-cert`: Path to SSL cert.
 - `ssl-key`: Path to SSL key.
 - `service-key`: Service key (used for accessing wallet system only).
 - `api-key`: API key (used for accessing all node APIs).
-- `wallet-auth`: Enable token auth for wallets (default: false).
-- `no-auth`: Disable auth for API server and wallets (default: false).
 - `cors`: Enable "Cross-Origin Resource Sharing" HTTP headers (default: false).
 
 Note: For security `cors` should not be used with `no-auth`.\
 If enabled you should also enable `wallet-auth` and set `api-key`.
 
-## Sample Config File
+## Wallet options
+
+These options must be saved in `wallet.conf`:
+
+### bcoin client:
+
+- `node-host`: Location of bcoin node HTTP server (default: localhost).
+- `node-port`: Port of bcoin node HTTP server (default: 80).
+- `node-ssl`: (bool) Whether to use SSL.
+- `node-api-key`: API-key for bcoin HTTP server.
+
+### Wallet database:
+
+- `memory`: Keep database in memory rather than write to disk.
+- `max-files`: Max open files for leveldb.
+- `cache-size`: Size (in MB) of leveldb cache and write buffer
+- `witness`: Make SegWit enabled wallets.
+- `checkpoints`: Trust hard-coded blockchain checkpoints.
+- `wipe-no-really`: Wipe the txdb - NEVER USE!
+
+### Wallet http server:
+
+- `ssl`: (bool) Whether to use SSL.
+- `ssl-key`: Path to SSL key.
+- `ssl-cert`: Path to SSL cert.
+- `http-host`: HTTP host to listen on (default: 127.0.0.1).
+- `http-port`: HTTP port to listen on (default: 8334 for mainnet).
+- `api-key`: API key (used for accessing all wallet APIs).
+- `wallet-auth`: Enable token auth for wallets (default: false).
+- `no-auth`: Disable auth for API server and wallets (default: false).
+- `cors`: Enable "Cross-Origin Resource Sharing" HTTP headers (default: false).
+- `admin-token`: Token required if `wallet-auth` is enabled.
```

Comment:
Could use an explanation that admin token is for restricting access to all wallet admin routes.